app/models/survey/response_option.rb

```diff
@@ -4,16 +4,6 @@ class Survey::ResponseOption < ApplicationRecord
   belongs_to :question, class_name: "Survey::Question", optional: true, foreign_key: "survey_question_id"
 
   store :title, coder: JSON
-
-  before_destroy :any_votes_present?, prepend: true do
-    throw(:abort) if errors.present?
```

Comment:
das hatte in meinen tests nicht funktioniert. es kam nie ein errors "oben" an, was in graphql zurückgegeben werden konnte.
mehr zeit sollten wir aber auch gar nicht reinstecken, da die anforderung ist, immer alles zu löschen:

> Wenn ich eine Umfrage lösche, löschen sich alle zugehörigen Antworten, Stimmen und Kommentare automatisch 

vorm löschen soll im CMS dann noch folgendes vorm löschen ausgegeben werden:

> Vor dem Löschen wird eine Bestätigung in einem Pop-Up abgefragt: "Möchten Sie diese Umfrage wirklich löschen? Damit werden auch alle zugehörigen Kommentare und Stimmabgaben gelöscht." + Optionen: "Abbrechen" und "Löschen"

die idee war aber sinnvoll und wird ja vllt. irgendwann ml benötigt. dann müssen wir aber noch etwas investieren, sodass es immer so funktioniert, dass bei einer destroy validierung auch die richtige meldung im graphql response durchgereicht wird.

Reply:
es geht bei dem any_votes_check ja auch um diesen Task hier: https://jira.tpwd.de/browse/SVA2-5

Als Verwaltungsmitarbeiter möchte ich nur Antworten löschen können, zu denen noch keine Abstimmungsdaten vorliegen, sodass Stimmen nicht verloren gehen.

Akzeptanzkriterien:

- Antwort kann gelöscht werden, wenn noch keine Stimmen abgegeben wurde
- Antwort kann nicht gelöscht werden, wenn schon Stimme abgegeben ist

Reply:
Also wenn eine Umfrage gelöscht werden soll, dann geht alles weg.
Aber einzelne Antworten können nur gelöscht werden, solange es keine Votes gibt.
Kriegen wir beide Anforderungen gleichzeitig auf Modelebene abgebildet?

Reply:
nein, bekommen wir nicht ohne immensen Aufwand, wir werden es im CMS UI abbilden